--- nodes copy.js	2015-09-17 13:56:30.000000000 +0800
+++ nodes.js	2015-09-17 19:35:47.000000000 +0800
@@ -3,8 +3,6 @@
  *  Functions for manipulating a JediScript AST.
  *  @author Joel Low <joel.low@nus.edu.sg>
  */
-define(function() {
-    'use strict';
     function is_null(xs) {
         return xs === null;
     }
@@ -77,7 +75,7 @@
         return is_object(object) && object.tag === tag;
     }
 
-    var module = {
+    module.exports = {
         stmt_line: function(stmt) {
             return stmt.line;
         },
@@ -326,11 +324,11 @@
         },
 
         eager_binary_expression: function(lhs, op, rhs, line_number) {
-            return module.application(module.variable(op, line_number), [lhs, [rhs, []]], line_number);
+            return this.application(this.variable(op, line_number), [lhs, [rhs, []]], line_number);
         },
 
         eager_unary_expression: function(op, expr, line_number) {
-            return module.application(module.variable(op, line_number), [expr, []], line_number);
+            return this.application(this.variable(op, line_number), [expr, []], line_number);
         },
 
         boolean_operation: function(lhs, op, rhs, line_number) {
@@ -531,5 +529,3 @@
         object_expression_pair_value: tail
     };
 
-    return module;
-});
